src/LuaBody.cpp

```diff
@@ -212,7 +212,10 @@ static int l_body_is_more_important_than(lua_State *l)
 	// bodies are sorted by importance in menus
 
 	if(body == other)
-		return false;
+	{
+		LuaPush<bool>(l, false);
+		return 1;
+	}
```

Comment:
This looks mostly to be a sanity check anyway, I'm almost of the opinion it should rather be an assert: it **seems** unlikely for this function to be called for the same pointers. What do you think?